### DISCUSSION TITLE
GROUPS POLICIES STORED ON A DIFFERENT PATH

### DISCUSSION THREAD

**Discussion by Markusx72** [Weatherlights/Winget-AutoUpdate-Intune#39] [Q&A]
Hi all, I distributed via Intune the MSI setup (1.04.0034.msi) but the Policies I configured with ADMX Template are not used. I see that policies are stored in the path HKEY_LOCAL_MACHINE\SOFTWARE\Policies\weatherlights.com\Winget-AutoUpdate but the Get-Policies function uses the path  HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Romanitho\Winget-AutoUpdate.
Do you know why?

**Comment by Weatherlights** [comment]
Hey... this release does not depend on the policies provided by the WAU project but builds on its own. Policies are refreshed by the Update which relies on the weatherlights.com\Winget-AutoUpdate path. This project released ADMX templates before the WAU project and has its own implementation with sometimes different approaches (like the modding feature)
